packages/empty-state/src/FeaturesEmptyState/FeaturesEmptyState.spec.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { ReactElement } from 'react';
+import { render } from '@testing-library/react';
+
+import Button from '@leafygreen-ui/button';
+import { Theme } from '@leafygreen-ui/lib';
+import { Link } from '@leafygreen-ui/typography';
+
+import { graphics } from '../example-graphics';
+
+import { MAX_NUM_FEATURES, MIN_NUM_FEATURES } from './FeaturesEmptyState';
+import { FeaturesEmptyState } from '.';
+
+type TestGraphics = Record<Theme, Array<ReactElement>>;
+const testGraphics = Object.keys(graphics).reduce((acc, theme) => {
+  acc[theme as Theme] = graphics[theme as Theme].map(
+    (graphic: any, index: number) => (
+      // jest will not process SVGs, so render them as <img /> elements
+      <img key={index} src={graphic} alt="" />
+    ),
+  );
+  return acc;
+}, {} as TestGraphics);
+
+const testFeatures = [
+  {
+    graphic: testGraphics[Theme.Light][0],
+    title: 'Multi-region, multi-cloud',
+    description:
+      'Run powerful and resilient apps that span multiple regions or clouds at once.',
+  },
+  {
+    graphic: testGraphics[Theme.Light][1],
+    title: 'Serverless and elastic',
+    description:
+      'Run powerful and resilient apps that span multiple regions or clouds at once.',
+  },
+  {
+    graphic: testGraphics[Theme.Light][2],
+    title: 'Always-on security',
+    description:
+      'Secure data with built-in defaults for access and end-toend encryption.',
+  },
+];
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('packages/empty-state/features', () => {
+  test('renders title', () => {
+    const { getByText } = render(
+      <FeaturesEmptyState title="test title" features={testFeatures} />,
+    );
+    expect(getByText('test title')).toBeInTheDocument();
+  });
+
+  test("console errors when there's only one feature", () => {
+    expect(MIN_NUM_FEATURES === 2).toBe(true);
```

Comment:
I'm a little confused by this line -- do we need to test this?

Reply:
It's just to make sure that the test case is relevant to the value of this variable. We could also just generate `MIN_NUM_FEATURES - 1` features every time, but I figured this was easier because I'd love to get this component out today.